src/main/kotlin/wordle/domain/Position.kt

```diff
@@ -0,0 +1,8 @@
+package wordle.domain
+
+class Position(private val position: Int) {
+
+    fun percent(value: Int): Int {
+        return position % value
+    }
```

Comment:
```suggestion
    fun percent(value: Int): Int = position % value
```

해당 부분도 문(statement) 보다 식(expression)을 사용하면 코드가 간결해집니다.